udp-rsc-offload.md

```diff
@@ -27,6 +27,9 @@ This section describes necessary updates in the Windows network stack to support
 - If hardware URO is not available, but a socket opts-in to URO with the Winsock API, software URO will be used for that socket. Software coalescing will follow the same rules as NDIS, except for the following:
     - Software coalescing may decide to use multiple NBLs in a chain of up to 255, instead of a single NBL.
     - Software coalescing will not coalesce IP fragments, broadcast or multicast datagrams, or datagrams with IP extension headers.
+    - Software coalescing will not coalesce UDP checksum validation failed
```

Comment:
```suggestion
    - Software coalescing will not coalesce failed UDP checksum validation
```
Though, I'm not sure this line is strictly necessary, because according to the text above `Software coalescing will follow the same rules as NDIS, except for the following:` and then below, in the NDIS section: `The UDP checksums on pre-coalesced packets must be correct.`

Reply:
I agree, we do not need to duplicate the NDIS rules here. If we do duplicate some of the rules, we should duplicate all of the rules for consistency. I'd lean towards documenting only the deviations from the general NDIS rules.